Step+XMLAdditions.h

```diff
@@ -0,0 +1,18 @@
+//
+//  Step+XMLAdditions.h
+//  DemoMonkey
+//
+//  Created by Nate Chandler on 3/15/15.
+//
+//
+
+#import "Step.h"
+
+extern NSString * __nonnull const StepXMLAdditionsKeyStep;
```

Comment:
Added macros to only use the nullability attributes when they're available in https://github.com/bignerdranch/DemoMonkey/commit/5d4a64f2b881a33fd605c0a7487cf9696a08de8f.  Used them in https://github.com/bignerdranch/DemoMonkey/commit/6550edb0cd532e3a19fd8ecbf8c00178b353b7ed.  It should compile with 6.2 and 6.3 now.